jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -1382,6 +1383,41 @@ public void setSessionTrackingModes(Set<SessionTrackingMode> sessionTrackingMode
                 _sessionHandler.setSessionTrackingModes(sessionTrackingModes);
         }
 
+        @Override
+        public int getSessionTimeout()
+        {
+            if (!isStarting())
+                throw new IllegalStateException();
+            if (!_enabled)
+                throw new UnsupportedOperationException();
+
+            int timeout = -1;
+            if (_sessionHandler != null)
+            {
+                int maxInactive = _sessionHandler.getMaxInactiveInterval();
+                timeout = maxInactive / 60;
+            }
+
+            return timeout;
+        }
+
+        @Override
+        public void setSessionTimeout(int sessionTimeout)
+        {
+            if (!isStarting())
+                throw new IllegalStateException();
+            if (!_enabled)
+                throw new UnsupportedOperationException();
+
+            if (_sessionHandler != null)
+            {
+                //sessionTimeout is in mins, but session and SessionHandler use seconds,
+                //both represented as an int, thus there is a floor and ceiling value
+                //for session timeout minutes.
+                _sessionHandler.setMaxInactiveInterval(SessionHandler.clipSessionTimeout(sessionTimeout) * 60); //in seconds
```

Comment:
No need to call a method in `SessionHandler` when it can be in this class.
This will keep the logic of dividing by `60` and multiplying by `60` in the same class.